libqtile/config.py

```diff
@@ -620,10 +620,11 @@ def match_func(value):
             if _type == 'title':
                 value = client.name
             elif _type == 'wm_class':
-                value = None
-                _value = client.window.get_wm_class()
-                if _value and len(_value) > 1:
-                    value = _value[1]
+                value = client.window.get_wm_class()
+                _match_func = match_func
+
+                def match_func(values):
```

Comment:
I'm confused, doesn't seem like this would work? Don't you want to do something like `def _match_func(...)...; match_func = _match_func`?

Reply:
I tested it again and it seems to work, with it I can match "firefox" or "Navigator" and both work, what is the concern with it?